### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by AustinStewart95**
Added end to end testing for add assignment feature.

### CODE REVIEW

File: cst438_gradebook/src/test/java/com/cst438/EndToEndTestAddAssignment.java [hunk 1]
```diff
@@ -66,35 +68,18 @@ public class EndToEndTestAddAssignment {
 	AssignmentRepository assignmentRepository;
 
 	@Test
-	public void addCourseTest() throws Exception {
```
Comment (MEGarb) [inline cst438_gradebook/src/test/java/com/cst438/EndToEndTestAddAssignment.java]: Deleting addCourseTest replacing with addAssignmentTest.  Is this what you intended?

Comment (AustinStewart95) [reply cst438_gradebook/src/test/java/com/cst438/EndToEndTestAddAssignment.java]: Yes, I copied over the default test case the professor gave and edited all the necessary fields for it to work when adding an assignment.

Comment (MEGarb) [reply cst438_gradebook/src/test/java/com/cst438/EndToEndTestAddAssignment.java]: I should have been more clear.  I dont see a problem with editing the file.  The problem I see is removing a unit test for a function that is still in use.  Did the professors instructions say to overwrite the test that was there?  If not I would suggest to readd that test.

Comment (AustinStewart95) [reply cst438_gradebook/src/test/java/com/cst438/EndToEndTestAddAssignment.java]: The add course end to end test still exists, but within a file named "EndToEndTestAddCourse". I simply used copied it as a template.

Comment (MEGarb) [reply cst438_gradebook/src/test/java/com/cst438/EndToEndTestAddAssignment.java]: I did not see that the test existed in another file.  Sorry for the confusion.

**Pull request merged**